Unity2DPlatformer/Assets/Scripts/PlayerController.cs

```diff
@@ -269,6 +272,6 @@ void OnCollisionEnter2D(Collision2D collision)
 
     void Die()
     {
-
+        rb.position = new Vector2(-5f, 6.11f);
```

Comment:
PlayerController is shared between levels, so there cannot be fixed coordinates. Please find "how to read player start position" and save it as variable in method Start() above. Then read those coordinates in in method Die()

Or if there is no such option (there should be - use google) use:

        string sceneName = SceneManager.GetActiveScene().name;
        int worldId = int.Parse(sceneName.Substring(6)); // Substring Level_<level_number>

to read world Id (levelId) and prepare switch-case for 4 levels with fixed coordinates (dirty solution, but will work, but Team not happy with this solution 😢).